index.js

```diff
@@ -38,10 +38,11 @@ module.exports = {
     'prefer-template': 'warn',
     'react/prop-types': 'error',
     'react-hooks/exhaustive-deps': 'error',
+    'comma-dangle': ['error', 'always-multiline'],
```

Comment:
this rule exists in v1 config
funny that `prettier` has this rule on, standard turns it off, we turn it back on

Reply:
I mean not like "ahahahaha funny" but still